### ISSUE TITLE
'Home page' link doesn't work on the 'end of project' celebration screen

### ISSUE THREAD

**Issue by mialondon** [LibCrowds/libcrowds#657]
Also, the message say 'e.title is undefined' - does it need a default if not supplied message?

**Comment by mialondon** [comment]
Actually, no links work! Firefox on work laptop...

**Comment by mialondon** [comment]
Also links don't work on Chrome (work laptop), and this time the error message is 'Cannot read property 'split' of undefined'.

**Comment by alexandermendes** [comment]
So it sounds like this is actually the "Home" button not working on the error page - the celebration screen doesn't have a home button, it should look something like this.

![image](https://user-images.githubusercontent.com/5636273/36818666-fbd5811e-1cdd-11e8-8b7b-104d6e591c18.png)

**Comment by mialondon** [comment]
Should we close things that aren't deployed yet?

**Comment by alexandermendes** [comment]
Yep I think they should be closed as the code is actually fixed and they can be checked on the staging server, otherwise it's very difficult to look down the list of issues and pick out the next thing to work on. Currently we would have 50 additional issues or so polluting the list where the work has actually been done, we just happen to not be running the latest version on our live server.

**Comment by mialondon** [comment]
But then I have to maintain a separate list of issues, just to check that things have been fixed. I know definitions of 'done' are tricky, but mine is that the technical work is done, the user experience is acceptable, and that the fix/feature is live.

**Comment by alexandermendes** [comment]
I would mostly agree, apart from something being live on a particular server. If that was the case we would need to reopen 100+ issues that have been closed since we last released a new version. It quickly gets out of hand and you would probably find you don't actually want to check and close them all!

How about I start mentioning you in the more frontendy things (e.g. this celebration screen) and get the ok to close, or ask you to close? And I carry on closing the more technical things backend things (e.g. migrate to the new version of x plugin, fixing errors etc.).

**Comment by alexandermendes** [comment]
There is also an aspect of wanting to finish things off. It gets a bit overwhelming when there are 100 issues in the queue and it's me that has to find time to tackle them all. So when I think that something has definitely been completed then it's nice to take it off the list. Still so much to do!

**Comment by mialondon** [comment]
Thanks for your thoughts! We should probably save this for when you're back in the office, but for, I agree we need a better process for things that have an immediate effect on the user experience (including curators as users) so it's not just 'this is technically complete' but 'this is holistically complete'. Let's chat about ways to manage the size of the apparent queue and the need to keep track of user-facing work. Mentions with a request for review/comment are probably a good place to start.

**Comment by alexandermendes** [comment]
Cool, yep let's talk next week!
